src/diagonal.works/b6/api/functions/features.go

```diff
@@ -55,6 +55,7 @@ func areaContainsAnyPoint(area b6.AreaFeature, points []s2.Point) (s2.Point, boo
 	return s2.Point{}, false
 }
 
+/*
```

Comment:
did u wanna delete this block

Reply:
if yes u also have one in functions/s2.go

Reply:
Ah yes, I meant to delete both. Done. Thanks for spotting!